docs/source/raft_configuration.md

```diff
@@ -231,35 +217,24 @@ channel, and then start the Raft instance for that chain.
 After it has successfully done so, the channel configuration can be updated to
 include the endpoint of the new Raft orderer.
 
-Removing a node from a Raft cluster is done by:
+To remove an ordering node from a channel:
+
+  1. Remove its endpoint from the channel config for the channel.
+  2. Remove its entry (identified by its certificates) from the channel configuration. For more information, check out [Add or remove orderers from existing channels](../create_channel/create_channel_participation.html#add-or-remove-orderers-from existing-channels).
```

Comment:
For this step, are you referring to removing it from the consenter set? This line is really vague and I think more specificity would help. You also include the link to the osnadmin remove section but don't really provide a recommendation that you should consider removing the channel from the orderer using the command.

Reply:
Yeah, I didn't go far enough here in making these instructions fully compatible with the new process. It's not really two steps, it's one. So I've changed this to: 

To remove an ordering node from the consenter set of a channel, use the `osnadmin channel remove` command to remove its endpoint and certificates from the channel. For more information, check out [Add or remove orderers from existing channels](../create_channel/create_channel_participation.html#add-or-remove-orderers-from-existing-channels).